standalone/ansible/all.yml

```diff
@@ -0,0 +1,6 @@
+
+---
+- import_playbook: setup.yml
+- import_playbook: postgres.yml
+- import_playbook: redis.yml
```

Comment:
Let's ensure redis is only accessible from the `app/sidekiq` servers and nowhere else.

Reply:
Added in `roles/redis.yml`